src/tests/fetch-all-rows.test.ts

```diff
@@ -3,13 +3,13 @@ import { Connection, DuckDB } from "@addon";
 const jsonResult = {
   bigint_col: 0n,
   bool_col: true,
-  date_string_col: "03/01/09",
+  date_string_col: Buffer.from("03/01/09"),
   double_col: 0,
   float_col: 0,
   id: 4,
   int_col: 0,
   smallint_col: 0,
-  string_col: "0",
+  string_col: Buffer.from("0"),
```

Comment:
Why aren't these strings anymore? A change in the parquet reader?

Reply:
The original files store that data as binary without UTF-8 encoding, meaning it was never an actual string